### PR TITLE
PR for autogenerating release notes

### PR DESCRIPTION
**Pull request by stepanLav**
That PR add workflow which will automatically create PR to add new version and release notes to repository - [nova-wallet-android-releases](https://github.com/novasamatech/nova-wallet-android-releases)
Example of autogenerated PR - https://github.com/novasamatech/nova-wallet-android-releases/pull/44

The Workflow will parse a comment from Pull Request and sent all data to another workflow - https://github.com/novasamatech/nova-wallet-android-releases/blob/main/.github/workflows/update-entrypoint.yml
Which will create or update Pull request with data from the comment

The workflow itself looks like:
Screenshot 2023-11-22 at 10 59 02

Format for the comment:
```
Release severity: Major | Critical | Normal
Release time: YYYY-MM-DDTHH:MM:SSZ
Release notes:

```
Example of comment:
https://github.com/novasamatech/nova-wallet-android/pull/1223#issuecomment-1822212003

Success run:
https://github.com/novasamatech/nova-wallet-android/actions/runs/6954020245/job/18920107429

Failed run:
https://github.com/novasamatech/nova-wallet-android/actions/runs/6953907205/job/18919797526

Screenshot 2023-11-22 at 11 01 10

### CODE REVIEW

**Comment by stepanLav** [comment]
Release notes:
Boost your DOT with Nova Wallet in our latest v7.3.0 release! 🚀

You can get more out of your previously crowdloanded DOT with Nova Wallet's Super Simple Staking! You can start securely staking with as little as 1 DOT and earn maximized rewards that pay out daily!

- disable Liquid Crowdloan Acala and Parallel API - users now can track lcDOT/cDOT directly in the Assets List
- fix tokens amount sync on the assets search

File: .github/workflows/pr_workflow.yml [hunk 1]
```diff
@@ -25,19 +30,69 @@ jobs:
...
+
+            if not time or not severity:
+              print(f"Current values: Time - '{time}', Severity - '{severity}'")
+              print("Time or severity cannot be empty, please set it in the PR comment with lines 'Release time:' and 'Release severity:'")
+              sys.exit(1)
+
+            with open(os.getenv('GITHUB_ENV'), 'a') as f:
+                f.write(f'TIME={time}\n')
+                f.write(f'SEVERITY={severity}\n')
+          parse_base_params(os.getenv("COMMENT_LINK"))
```
Comment (valentunn) [inline .github/workflows/pr_workflow.yml:89]: I suggest to extract it into a separate .py file

**Review by valentunn — APPROVED** [review]
Can we auto-generate timestamp based on comment's timestamp?

File: .github/workflows/pr_workflow.yml (hunk 1 above)
Comment (stepanLav) [reply .github/workflows/pr_workflow.yml]: extracted to separate file, keep it in .github repo, checked on test rc/3.2.1

**Pull request merged**